sympy/solvers/ode/tests/test_systems.py

```diff
@@ -594,33 +630,33 @@ def test_sysode_linear_neq_order1_type1():
 
     # Regression test case for issue #14312
     # https://github.com/sympy/sympy/issues/14312
-    eq11 = (Eq(Derivative(x(t),t), k3*y(t)), Eq(Derivative(y(t),t), -(k3+k2)*y(t)), Eq(Derivative(z(t),t), k2*y(t)))
+    eq11 = (Eq(Derivative(x(t), t), k3*y(t)), Eq(Derivative(y(t), t), -(k3 + k2)*y(t)), Eq(Derivative(z(t), t), k2*y(t)))
     sol11 = [Eq(x(t), C1 + C3*k3*exp(t*(-k2 - k3))/k2), Eq(y(t), -C3*(k2 + k3)*exp(t*(-k2 - k3))/k2),
             Eq(z(t), C2 + C3*exp(t*(-k2 - k3)))]
     assert dsolve(eq11) == sol11
     assert checksysodesol(eq11, sol11) == (True, [0, 0, 0])
 
     # Regression test case for issue #14312
     # https://github.com/sympy/sympy/issues/14312
-    eq12 = (Eq(Derivative(z(t),t), k2*y(t)), Eq(Derivative(x(t),t), k3*y(t)), Eq(Derivative(y(t),t), -(k3+k2)*y(t)))
+    eq12 = (Eq(Derivative(z(t), t), k2*y(t)), Eq(Derivative(x(t), t), k3*y(t)), Eq(Derivative(y(t), t), -(k3 + k2)*y(t)))
     sol12 = [Eq(z(t), C1 - C3*k2*exp(t*(-k2 - k3))/(k2 + k3)), Eq(x(t), C2 - C3*k3*exp(t*(-k2 - k3))/(k2 + k3)),
             Eq(y(t), C3*exp(t*(-k2 - k3)))]
     assert dsolve(eq12) == sol12
     assert checksysodesol(eq12, sol12) == (True, [0, 0, 0])
 
     # Regression test case for issue #15474
     # https://github.com/sympy/sympy/issues/15474
-    eq13 = [Eq(diff(f(t), t), 2 * f(t) + g(t)),
-         Eq(diff(g(t), t), a * f(t))]
+    eq13 = [Eq(diff(f(t), t), 2*f(t) + g(t)),
+            Eq(diff(g(t), t), a*f(t))]
     sol13 = [Eq(f(t), -C1*exp(t*(1 - sqrt(a + 1)))/(sqrt(a + 1) + 1) + C2*exp(t*(sqrt(a + 1) + 1))/(sqrt(a + 1) - 1)),
             Eq(g(t), C1*exp(t*(1 - sqrt(a + 1))) + C2*exp(t*(sqrt(a + 1) + 1)))]
 
     assert dsolve(eq13) == sol13
     assert checksysodesol(eq13, sol13) == (True, [0, 0])
 
-    eq14 = [Eq(f(t).diff(t), 2 * g(t) - 3 * h(t)),
-           Eq(g(t).diff(t), 4 * h(t) - 2 * f(t)),
-           Eq(h(t).diff(t), 3 * f(t) - 4 * g(t))]
+    eq14 = [Eq(f(t).diff(t), 2*g(t) - 3*h(t)),
```

Comment:
Did you intend to change these spaces or is that your editor doing that?

Reply:
I prefer the new spacing but I just wanted to check that it is something you are doing intentionally because it seems many unrelated tests are changed in this PR.

Reply:
> Did you intend to change these spaces or is that your editor doing that?

I changed it intentionally, as in the previous PR you mentioned how these unwanted spaces made the examples look bad.